core/src/main/java/nu/yona/server/messaging/entities/MessageDestination.java

```diff
@@ -5,21 +5,16 @@
 package nu.yona.server.messaging.entities;
```

Comment:
This is because I am caching `MessageDestinationDTO` instances. These should not contain the set of messages. The `MessageService` should be able to use the message destination using a `MessageDestinationDTO`.

I didn't like the `MessageDestination` structure at all. It required `MessageDestination.getRepository().save(messageDestination)` every time when sending a message.